packages/mui-codemod/src/v6.0.0/styled/test-cases/ObjectMap.expected.js

```diff
@@ -4,8 +4,6 @@ const IconRoot = styled('span')(({
   userSelect: 'none',
   width: '1em',
   height: '1em',
-  // Chrome fix for https://bugs.chromium.org/p/chromium/issues/detail?id=820541
```

Comment:
Is this not relevant anymore?

Reply:
We need to keep track of the status of each of these bugs. As soon as they are fixed an available in enough browser, we need to update our implementation. Now, this is a codemod file, I think removing this comment will make it easier to maintain those.